joomla/joomla-cms
# [4.0] Stop TinyMCE autochanging skins when new skins are uploaded

Redo of #22097 .

### Summary of Changes

Currently, we store TinyMCE skin setting as a numeric value provided by `glob()`. This can cause the skin to automatically change when uploading a new skin.
This removes custom `JFormFieldSkins` field and replaces it with standard folder list field in form.

### Testing Instructions

Edit TinyMCE plugin. In your set (Set 0 by default) select a skin. Save the plugin.
Upload a skin with name that alphabetically orders before your selected skin.
Edit an article.
You should see the new skin being used or the editor not appearing at all.
Remove the custom skin.
Apply patch.
Save plugin configuration again.
Upload the skin again.
Edit an article.
Now your selected skin should be used.

### Documentation Changes Required

`JFormFieldSkins` is removed.
